src/main/steps/common/confirm-contact-details/address/lookup/form.njk

```diff
@@ -0,0 +1,9 @@
+{% extends "common/form/form.njk"%}
+
```

Comment:
Duplicate file to one we have under "src/main/steps/applicant/confirm-contact-details/address/lookup/form.njk"

Reply:
Will complete refactoring as part of follow up ticket - other refactoring required throughout the different journeys